nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-services-api/pom.xml

```diff
@@ -31,5 +31,23 @@
             <groupId>com.google.auth</groupId>
             <artifactId>google-auth-library-oauth2-http</artifactId>
         </dependency>
+        <!-- please do not remove grpc-* dependencies because they are used by GCP processors at runtime,
```

Comment:
can ditch this comment in favor of listing the intended scope with the dependencies